src/pages/lessons/[slug].tsx

```diff
@@ -222,16 +229,17 @@ const Lesson: FunctionComponent<LessonProps> = ({initialLesson}) => {
     }
   }
 
-  const completeVideo = () => {
+  const completeVideo = (lessonView: any) => {
     if (lessonView) {
-      const progress = getProgress()
-      if (progress?.rate_url) {
+      const hasNextLesson = nextLesson || nextUp.nextUpPath
+      const progress = getProgress(lessonView)
+      if (!hasNextLesson && progress?.rate_url) {
```

Comment:
we are only gonna pop the rating if it's the last lesson and there is no place else to go